### ISSUE TITLE
Electrum Rust Server running but Electrum wallet is "not connected"

### ISSUE THREAD

**Issue by Nerfzooka** [raspiblitz/raspiblitz#2513]
Issue #1895 is happening again.  
Raspiblitz v 1.7.0 running 3 services:  BTCPayServer, BTC-RPC-Explorer and Electrum Rust Server  can explore blockchain and transactions but my Electrum wallet on the same network says"not connected".
Tried rebooting, reindexing, restarting, no change.  Double checked that the Electrum wallet automatic server selection is off and the rust ip and port are right.  Nginx is running the BTC-RPC and BTCPayServer websites fine. I can ping the raspiblitz from the wallet computer and ping the wallet computer from the raspiblitz.  Here's a weird observation:  the Electrum wallet block count is 2 higher than the raspiblitz.
![raspiblitz](https://user-images.githubusercontent.com/23216537/132110798-b2a1bc7c-dc69-4610-b203-6cc50fc535f5.jpg)

**Comment by rootzoll** [comment]
Can you post the status report you get when in the RaspiBlitz menu you go `ELECTRS` and then `STATUS`?

**Comment by Nerfzooka** [comment]
Yep
![electrsStatus](https://user-images.githubusercontent.com/23216537/134262897-b8924e79-a876-4ac9-b02b-3f2bdad96dff.png)
.

**Comment by walletofsatoshi** [comment]
Same issue. It worked fine earlier now exact same issue. Cant connect electrum but can connect bluewallet over Lan on port 50001. Strange

**Comment by rootzoll** [comment]
Next week the v1.7.1 will get released with an updated Electrs ... please try with that updated version and report in if the problem still persists.

**Comment by Nerfzooka** [comment]
While we're waiting for the update, I was looking at my BTC-RPC explorer node status. It sees a problem with Data Indexes.   I reindexed, but no change.    hmmmm

![rustStatus](https://user-images.githubusercontent.com/23216537/137418362-09597a90-85c0-4048-8f92-1e9c3c172870.png)

**Comment by Nerfzooka** [comment]
Updated to 1.7.1 and have a slightly different problem.  Electrum wallet still says "not connected", but BTC-RPC Explorer is also not working now with a "502 Bad Gateway".  On the bright side the BTCPay server is serving its web page, so nginx must be working.

![info](https://user-images.githubusercontent.com/23216537/137649052-65cf9265-8712-42ff-93a4-4dc394ea4f05.png)

**Comment by Nerfzooka** [comment]
Hmm... Is electrs still indexing?

![rustStatus](https://user-images.githubusercontent.com/23216537/137650551-ae35040e-8417-496e-8eb8-35a17b764d55.png)

**Comment by Nerfzooka** [comment]
I guess it was. Now electrs is synced and BTC-RPC explorer is working, but electrum wallet still says "not connected"
![electrsStatus](https://user-images.githubusercontent.com/23216537/137779789-bf344738-a1dc-4ba0-9f8b-e90e07c4e1a7.png)
.

**Comment by Nerfzooka** [comment]
Hey, Electrum wallet is connected after I rebooted the raspiblitz to remove lcd screen .  👍

**Comment by Nerfzooka** [comment]
Electrum wallet connected to another server, not mine.  I thought it was fixed for a minute, but it's not.

**Comment by rootzoll** [comment]
@openoms have you heard of similar reports in the support chats?

**Comment by openoms** [comment]
No issues after the reindexing finished after the the update to v1.7.1.
If there are issues connecting despite no errors in:

`sudo journalctl -n100 -u electrs`

try deleteing the electrum certificates on the desktop.

**Comment by Nerfzooka** [comment]
sudo journalctl ... created 65 pages of this:
![journalctl](https://user-images.githubusercontent.com/23216537/138739717-20502071-b149-4558-9bb2-66612050e5be.png)
It looks like all chain updating and indexing, but I don't' know how to study it more closely

I don't know what or where any electrum certificates are so I uninstalled and installed electrum on the desktop.  It still says "not connected".

**Comment by openoms** [comment]
use the exact command above to show only the last 100 lines

**Comment by Nerfzooka** [comment]
Thanks for the help, @openoms ! The last 100 lines of journalctl have no errors, just chain updates and block indexing.

**Comment by Nerfzooka** [comment]
Ah, I need electrum command line to delete certificates.  Is there a way to do that on windows?

**Comment by openoms** [comment]
@Nerfzooka  just empty the certs from User/appdata/roaming/electrum/certs/

**Comment by Nerfzooka** [comment]
Hey @openoms , I'm so happy, Clearing out the cert folder fixed the problem.   Yay!

**Comment by lupn3** [comment]
Raspiblitz 1.8.0 running, full synced

Electrs
- is activated
- has opened port 50002
- is connected to localhost

but suddenly out of the nowhere the connection to the wallet isn't possible anymore. It seems that Electrs is receiving the request from the wallet (not: electrum wallet), but not responding to it, even after reinstall an re-indexing Electrs.

**Comment by Nerfzooka** [comment]
Oh no, @lupn3, after updating Raspiblitz to 1.8.0 I've got the same problem again too.  
Electrs is running, All is synced, All is indexed. BTC-RPC-Explorer is running normally, but electrum wallet is not connected.  The wallet syncs immediately if I choose another server, but shows no connection when looking at the raspiblitz.  Emptying the electrum wallets cert folder didn't work this time.
![ElectrsStatus 2](https://user-images.githubusercontent.com/23216537/196848632-fb7e8085-2af7-47b8-a5d8-009d60ffd648.png)

**Comment by lupn3** [comment]
@Nerfzooka Have opened an additional issue here (https://github.com/rootzoll/raspiblitz/issues/3343). Unfortunately, I still have not made any progress. Superficially everything looks good, but the Raspi still refuses any connection to the wallet. It is exasperating.

**Comment by Nerfzooka** [comment]
TLDR: This was fixed for me by uninstalling then reinstalling electrs on the Raspiblitz and deleting electrum wallet certificates from the desktop.
